packages/react-core/src/components/Slider/Slider.tsx

```diff
@@ -474,7 +474,12 @@ export const Slider: React.FunctionComponent<SliderProps> = ({
           </div>
         )}
         {hasTooltipOverThumb ? (
-          <Tooltip triggerRef={thumbRef} entryDelay={0} content={findAriaTextValue()}>
+          <Tooltip
+            className={css('.pf-v6-m-tabular-nums')}
```

Comment:
It shouldn't have the `.` in the classname should it?

Also is there a way we could access the class on a react-styles object instead of hardcoding it?

Reply:
Cut and paste error.  I will fix it.
There does not seem to be an object for that one.  @mcoker should this be added to the Slider stylesheet?  Or perhaps added to utilities?

Reply:
This is one of only a few global modifier classes in core, it's like a utility class but it ships with PF by default, so we and users can use it in components and examples and things and not have to import any utility classes. Since this is a built-in feature of the slider, I don't think we should use a utility class since that will require importing a separate stylesheet to use it.

If we wanted to add a style somewhere for this, off the top of my head, we might add it in one of a couple of places:
* As a variation on the tooltip component (like `.pf-v6-c-tooltop.pf-m-tabular-nums` or something) and we would write a style rule for the tooltip content to use tabular numbers.
* As a style of the slider, like `.pf-v6-c-slider__tooltip-value`, and you would wrap the children (`{thumbComponent}`) in this tooltip in `<span class="pf-v6-c-slider__tooltip-value">{thumbComponent}</span>`

IMO the global modifier is ideal because there may be a handful of places where we just want to add this single style, and it avoids having to duplicate the style in a bunch of components. But I don't feel strongly about it, and can see the utility in having a style in the slider component that you could pass to the tooltip (the second bullet above). That lets us bundle other styles there for that value if we need to, and we can update the styling (say to remove the tabular numbers for some reason), and it's just a CSS change and not a markup change, where the markup change may be more breaking than a CSS change (if tests fail because of the markup change, for example).

All that said, wdyt about the global modifier versus either of those options?

Reply:
So the global modifier option would leave the core implementation as is correct?

Reply:
@tlabaj yep 👍